spec/defines/jenkins_plugin_spec.rb

```diff
@@ -271,6 +271,9 @@ class { jenkins:
     describe 'pin parameter' do
       context 'with pin => true' do
         let(:params) { { pin: true } }
+        let :pre_condition do
```

Comment:
Since you include it inside the plugin class, is this precondition still needed?

Reply:
fixed